lambda/moviebotFunction.js

```diff
@@ -236,6 +236,21 @@ function unsureResult(intentRequest, callback) {
     callback(elicitIntent(sessionAttributes, getFurtherInfoMessage(sessionAttributes)))
 }
 
+// TODO do a better way
+function updateSlotWithLiteralValue(intentRequest) {
+    if (intentRequest.currentIntent.name == 'FindMovieByQuote') {
+        if (intentRequest.invocationSource == 'FulfillmentCodeHook') {
+            intentRequest.currentIntent.slots.MovieQuote = intentRequest.inputTranscript.replace(/\\'/, "'");
+        }
+    }
+    if (intentRequest.currentIntent.name == 'FindMovieByPlot') {
+        if (intentRequest.invocationSource == 'FulfillmentCodeHook') {
+            intentRequest.currentIntent.slots.PlotDescription = intentRequest.inputTranscript.replace(/\\'/, "'");
+        }
```

Comment:
the inputTranscript might not always guarantee to be `PlotDescription` or `Quote` since we match utterance like "The movie was about {PlotDescript} and The movie has {MovieQuote}​ quote in it